corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js

```diff
@@ -262,9 +262,15 @@ PhoneEntry.prototype.constructor = FreeTextEntry;
 function FloatEntry(question, options) {
     IntEntry.call(this, question, options);
     this.templateType = 'str';
+    this.lengthLimit = options.lengthLimit || 15;
+    var valueLimit = options.valueLimit || +("9".repeat(14));
 
     this.getErrorMessage = function (rawAnswer) {
-        return (isNaN(+rawAnswer) ? "Not a valid number" : null);
+        if (isNaN(+rawAnswer))
+            return "Not a valid number";
```

Comment:
Not related to this PR, but shouldn't these be translated?

Reply:
Hm, probably. I think there are a bunch of things in formplayer of which that is true